src/main/java/seedu/address/logic/parser/DeleteMultipleCommandParser.java

```diff
@@ -48,74 +48,81 @@ public DeleteMultipleCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_NAME).isPresent()
                 && !argMultimap.getValue(PREFIX_NAME).get().trim().isEmpty()) {
             List<String> nameKeywords = argMultimap.getAllValues(PREFIX_NAME);
-            int numOfEmptyValue = nameKeywords.stream()
-                    .filter(item-> item.isEmpty()).collect(Collectors.toList()).size();
+            int numOfEmptyValue = (int) nameKeywords.stream()
+                    .filter(String::isEmpty).count();
             if (numOfEmptyValue != 0) {
                 throw new ParseException(
-                        String.format(MESSAGE_INVALID_COMMAND_FORMAT, DeleteMultipleCommand.MESSAGE_USAGE));
+                        String.format(MESSAGE_INVALID_COMMAND_FORMAT_EMPTY_VALUE, "Name",
```

Comment:
again, I think theres no need to capitalise N in name as the user does not know it is a class!